conda/activate.py

```diff
@@ -203,7 +203,7 @@ def raise_invalid_command_error(actual_command=None):
             raise_invalid_command_error()
 
         command = arguments[0]
-        arguments = tuple(drop(1, arguments))
+        arguments = tuple(arguments[1:])
```

Comment:
```python
command, *arguments = arguments
```

Reply:
The old code seemed really excited about keeping everything as a tuple()

Reply:
yeah, I noticed, I guess leaving it as a tuple provides some level of guarantee 🤷🏼‍♂️

but in this case `arguments` is only a local var and isn't passed to another function, so I see very little potential for dangerous mutations